server/misc/miscCode.js

```diff
@@ -32,3 +32,125 @@ for (const dataPoint in excelObj) {
   }
   prevRow = currRowNumber;
 }
+
+res.locals.output = [
+  {
+    tableName: 'people',
+    columns: [
+      { name: 'VARCHAR(255)' },
+      { mass: 'FLOAT' },
+      { hair_color: 'VARCHAR(255)' },
+      { skin_color: 'VARCHAR(255)' },
+      { eye_color: 'VARCHAR(255)' },
+      { birth_year: 'VARCHAR(255)' },
+      { gender: 'VARCHAR(255)' },
+      { height: 'INT' },
+    ],
+  },
+  {
+    tableName: 'species',
+    columns: [
+      { name: 'VARCHAR(255)' },
+      { classification: 'VARCHAR(255)' },
+      { average_height: 'VARCHAR(255)' },
+      { average_lifespan: 'VARCHAR(255)' },
+      { hair_colors: 'VARCHAR(255)' },
+      { skin_colors: 'VARCHAR(255)' },
+      { eye_colors: 'VARCHAR(255)' },
+      { language: 'VARCHAR(255)' },
+    ],
+  },
+  {
+    tableName: 'films',
+    columns: [
+      { title: 'VARCHAR(255)' },
+      { director: 'VARCHAR(255)' },
+      { producer: 'VARCHAR(255)' },
+      { release_date: 'DATE' },
+    ],
+  },
+];
+
+// res.locals.output = [
+//   {
+//     tableName: 'people',
+//     columns: [
+//       { name: 'VARCHAR(255)' },
+//       { mass: 'FLOAT' },
+//       { hair_color: 'VARCHAR(255)' },
+//       { skin_color: 'VARCHAR(255)' },
+//       { eye_color: 'VARCHAR(255)' },
+//       { birth_year: 'VARCHAR(255)' },
+//       { gender: 'VARCHAR(255)' },
+//       { height: 'INT' },
+//     ],
+//   },
+//   {
+//     tableName: 'species',
+//     columns: [
+//       { name: 'VARCHAR(255)' },
+//       { classification: 'VARCHAR(255)' },
+//       { average_height: 'VARCHAR(255)' },
+//       { average_lifespan: 'VARCHAR(255)' },
+//       { hair_colors: 'VARCHAR(255)' },
+//       { skin_colors: 'VARCHAR(255)' },
+//       { eye_colors: 'VARCHAR(255)' },
+//       { language: 'VARCHAR(255)' },
+//     ],
+//   },
+//   {
+//     tableName: 'films',
+//     columns: [
+//       { title: 'VARCHAR(255)' },
+//       { director: 'VARCHAR(255)' },
+//       { producer: 'VARCHAR(255)' },
+//       { release_date: 'DATE' },
+//     ],
+//   },
+// ];
```

Comment:
I believe this is the exact same as what we have above, correct? If so then let's remove the duplicate code.